### ISSUE TITLE
font changes don't really work

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/SymmetryBook#116]
Look at this:

<img width="586" alt="Screen Shot 2021-08-06 at 10 29 56 AM" src="https://user-images.githubusercontent.com/700228/128526192-c1cffed7-f3a2-43f2-93a0-29ac45e2f70d.png">

I think using a font change to indicate application of a function is too cute, and likely to cause confusion.

**Comment by bidundas** [comment]
Here I beg to differ. Using bold fonts to signify important constants has an enormous precedence even though it has some obvious disadvantages.  Personally I'd prefer to use a simple bold (and not black board bold). 

Bjorn

**Comment by DanGrayson** [comment]
Well, for constants, it might be okay.   But the example above is about the use for variables.

**Comment by UlrikBuchholtz** [comment]
The problem with a simple bold is that it's a bit harder to tell apart from the regular weight.

@DanGrayson , what would you prefer instead as the notation for the canonical finite set with n elements? Fin n?

**Comment by bidundas** [comment]
I think we should be pragmatic here. (Black board) Bold n is easy to read and only the most dogmatic of our readers will even think about it.  If you feel strongly about it, we could under- or over line instead, but please stay clear of [n] and n_+. 

Bjorn

**Comment by pierrecagne** [comment]
I'm not sure I understand the critic here. Is it about the differences between numerals and elements of N? In other words, do you mean that you would prefer having "For all numeral n, abbreviate \bn{n} [...]"? 

If not, could you be more explicit about what is bothering you?

**Comment by UlrikBuchholtz** [comment]
> And what about `\bn m`? Is it covered by this or not?

Yes, that's covered. But one case that isn't covered – and which we might need – is the case of a compound expression of type Nat. There we still need a way to map that to the corresponding canonical finite set.

**Comment by DanGrayson** [comment]
> > And what about `\bn m`? Is it covered by this or not?
> 
> Yes, that's covered.

How is the student supposed to know it's covered?
